udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -0,0 +1,456 @@
+                <p class="fr-m-0 fr-text--xs">{{ $t('On {site}, a dataset is a set of files.', {site: title}) }}</p>
+              </div>
+            </div>
+          </Well>
+          <fieldset class="fr-fieldset" aria-labelledby="description-legend">
+            <legend class="fr-fieldset__legend" id="description-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Description") }}
+              </h2>
+            </legend>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="nameDatasetAccordionId"
+              @blur="vWarning$.title.$touch"
+            >
+              <InputGroup
+                :aria-describedby="nameDatasetAccordionId"
+                :label="$t('Dataset name')"
+                :required="true"
+                v-model="dataset.title"
+                :hasError="stateHasError('title')"
+                :hasWarning="stateHasWarning('title')"
+                :errorText="getErrorText('title')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addAcronymAccordionId"
+              @blur="vWarning$.acronym.$touch"
+            >
+              <InputGroup
+                :label="$t('Acronym')"
+                v-model="dataset.acronym"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="writeAGoodDescriptionAccordionId"
+              @blur="vWarning$.description.$touch"
+            >
+              <InputGroup
+                :label="$t('Description')"
+                :required="true"
+                type="textarea"
+                v-model="dataset.description"
+                :hasError="stateHasError('description')"
+                :hasWarning="stateHasWarning('description')"
+                :errorText="getErrorText('description')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="useTagsAccordionId"
+              @blur="vWarning$.tags.$touch"
+            >
+              <MultiSelect
+                :minimumCharacterBeforeSuggest="2"
+                @change="(value) => dataset.tags = value"
+                :placeholder="$t('Tags')"
+                :searchPlaceholder="$t('Search a tag...')"
+                suggestUrl="/tags/suggest/"
+                :values="dataset.tags"
+                :hasWarning="stateHasWarning('tags')"
+                :multiple="true"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="selectLicenseAccordionId"
+              @blur="vWarning$.license.$touch"
+            >
+              <MultiSelect
```

Comment:
We had implemented [grouped dropdown in admin](https://github.com/opendatateam/udata/pull/2786). Do we want the groups as well here?

Reply:
We can add this, also we're not restricted on how to display them

Reply:
Tracked in https://github.com/etalab/data.gouv.fr/issues/1195